conf/dokuwiki.php

```diff
@@ -17,6 +19,7 @@
 $conf['start']       = 'start';           //name of start page
 $conf['lang']        = 'en';              //your language
 $conf['template']    = 'dokuwiki';         //see lib/tpl directory
+$conf['frontEndTemplate']    = '';         //frontend template to use for public pages
```

Comment:
don't understand
I want to keep backend (default) template but change frontend. so when i will create a frontend template, it will no affect admin page.